lib/ipaddr.mli

```diff
@@ -37,7 +37,6 @@ type scope =
 module V4 : sig
   (** Type of the internet protocol v4 address of a host *)
   type t [@@deriving sexp]
-  val compare : t -> t -> int
```

Comment:
These `compare` bindings are also exposed by the included Map type, hence the duplication is removed here